demo/simple/reljam-view.sysl

```diff
@@ -0,0 +1,44 @@
+# reljam view /reljam-View GroupModelToApi
+# TODO
+
+GoupModel [package="io.sysl.demo.view.model"]:
```

Comment:
GoupModel?

Reply:
Yeah, not idea, I agree. Maybe I will just remove the whole file.
For context: I wanted to create a simplified Petshop sample. So I thought I model a class in a school with students. Then I realised that the word class is quite overloaded in IT, so I changed it to group.
I will remove this.